src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,75 @@
+.menu {
+  background-color: $colorBackground;
+  
+  &__content {
+    box-sizing: border-box;
+    height: 100vh;
+    padding: $headerTopPading 0;
+  }
```

Comment:
Check the #17 of checklist. Your menu doesn't work good on the screens with small height, it should scroll down.